src/components/CommentInput/helpers.jsx

```diff
@@ -35,6 +35,16 @@ turndownService.addRule('taskListItems', {
   },
 })
 
+// if there is a double backslash followed by text, \\text, replace with \\\ so it resolves to \\text
```

Comment:
Was this comment lacking a backslash or is there a reason it's three?
```suggestion
// if there is a double backslash followed by text, \\text, replace with \\\\ so it resolves to \\text
```

Reply:
I don't see why it would be 4 slashes? The string `\\test` if being replaced with `\\\test` (3 \\\).

Reply:
Okay I see what you mean now. More a question of if `\\test` should be replaced with `\\\test` or `\\\\test`.

Reply:
Yup - I wonder what, if escaping only one what would happen with text like `\\n` becoming `\\\n` which technically would mean a backslash and a new line. Whereas `\\\\n` would mean two backslashes and the `n` character?

Reply:
I am a bit struggling here why does it matter since from what Luke wrote in either case (triple or quadruple backslash) it can be resolved back to `\\n`. Only case if you want make comment AND make a new line (?)

That being said, making it double and then diving it into half seems more reasonable approach... (so `\\` to `\\\\` and back  to `\\`).

Reply:
Okayyyyy fine 4 it is. But only because it makes this code even more silly `content.replaceAll('\\\\', '\\\\\\\\')`